common/src/main/scala/models/elections/ElectionResults.scala

```diff
@@ -0,0 +1,25 @@
+package models.elections
+
+import java.net.URI
+
+import models.JsonUtils
+import play.api.libs.json.Json
+
+case class CandidateResults(
+  name: String,
+  states: List[String],
+  electoralVotes: Int,
```

Comment:
Ahah for some reason I was missing three zeros for max_int. My bad, please ignore